### PR TITLE
Configure procs retries (on failure)

### LINKED ISSUE #25: Procs are rerun on failure
As an admin,
I want to configure the number of retries for procs on failures.

So that,
Users contributing to procs can have control if they haven't handled immutability, state, etc. A proc will be run once under all circumstances. On failure, it can be configured to not rerun.

Currently, it's retried 6 times on failure by default.

### PR DESCRIPTION
**Pull request by olttwa**
- Modify Changelog format

Resolves #25

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+### Modified
+  - Changelog format
+  - Logrus dependency path
+
+## v0.1.0 - 2018-10-08
+### Added
   - Add Authentication Headers: Email-Id and Access-Token
   - Add Job success and failure metrics
+
+[0.2.0]: https://github.com/gojektech/proctor/compare/v0.1.0...v0.2.0
```
Comment (achilles42) [inline CHANGELOG.md]: why this link?

File: proctord/config/config.go [hunk 2]
```diff
@@ -60,6 +60,11 @@ func KubeJobActiveDeadlineSeconds() *int64 {
 	return &tmp
 }
 
+func KubeJobRetries() *int32 {
+	tmp := int32(viper.GetInt("KUBE_JOB_RETRIES"))
```
Comment (achilles42) [inline proctord/config/config.go]: can we rename `tmp` to more meaningful?

File: proctord/glide.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 package: github.com/gojektech/proctor/proctord
 import:
-- package: github.com/Sirupsen/logrus
+- package: github.com/sirupsen/logrus
```
Comment (achilles42) [inline proctord/glide.yaml]: This breaks on case-sensitive disk format.

File: proctord/logger/logrus.go [hunk 4]
```diff
@@ -5,7 +5,7 @@ import (
 
 	"github.com/gojektech/proctor/proctord/config"
 
-	log "github.com/Sirupsen/logrus"
+	log "github.com/sirupsen/logrus"
```
Comment (achilles42) [inline proctord/logger/logrus.go]: This also breaks on case-sensitive disk format.

File: CHANGELOG.md (hunk 1 above)
Comment (olttwa) [reply CHANGELOG.md]: We're following a similar structure of CHANGELOG across all repos. It's inspired from this repo: https://github.com/olivierlacan/keep-a-changelog/blob/master/CHANGELOG.md

Comment (olttwa) [reply CHANGELOG.md]: This link will take you straight to a comparison with previous tag/release

File: proctord/config/config.go (hunk 2 above)
Comment (olttwa) [reply proctord/config/config.go]: It's actually a temporary variable. The only reason it exists is you can't instantiate a pointer literal in golang. You need to assign it to a variable and reference the variable to pass a pointer.

**Pull request merged**